src/container/oid_iv.c

```diff
@@ -79,6 +76,12 @@ oid_iv_ent_refresh(struct ds_iv_entry *iv_entry, struct ds_iv_key *key,
 	struct oid_iv_range	*oids;
 	struct oid_iv_range	*avail;
 
+	if (src == NULL) {
+		D_DEBUG(DB_TRACE, "delete entry iv_entry %p\n", iv_entry);
+		iv_entry->iv_to_delete = 1;
```

Comment:
What's this change for? Why it's only required for oid_iv_ent_refresh()?

Reply:
Oh, OID IV entry is a little different here, and each container will have a separate IV entry.  Since we can not delete the IV entry directly inside the container entry IV callback, so set a flag to tell the IV (engine/server_iv.c) to delete the entry once all processing (callback) are done.